contentcuration/contentcuration/models.py

```diff
@@ -621,9 +621,9 @@ def save(self, *args, **kwargs):  # noqa: C901
             # Delete db if channel has been deleted and mark as unpublished
             if not original_channel.deleted and self.deleted:
                 self.pending_editors.all().delete()
-                channel_db_url = os.path.join(settings.DB_ROOT, self.id) + ".sqlite3"
-                if os.path.isfile(channel_db_url):
-                    os.unlink(channel_db_url)
+                export_db_storage_path = os.path.join(settings.DB_ROOT, "{channel_id}.sqlite3".format(channel_id=self.id))
+                if default_storage.exists(export_db_storage_path):
+                    default_storage.delete(export_db_storage_path)
```

Comment:
This is unrelated to #1455 but I thought I'd fix this as well. No automated tests for it, but I exercised this code path locally and the .sqlite3 file indeed gets deleted.

Reply:
Can we get tests for this? Especially since it's a deletion codepath and it's called from Channel.save, it'd be good to confirm that it ONLY deletes if `deleted` is set. You can just create an empty db file in the proper location for the test. 

It might also be good to add a tech debt note that we probably shouldn't have a delete operation in save, and should move deletion to an external method that gets called by any deletion codepath. A bug that causes `deleted` to get set when it shouldn't would mean the db would be gone for possibly quite some time until we republish.

Reply:
Okay I can add tests for this.  Which file would you recommend as the best place to append to: `test_createchannel.py` or `test_channel_model.py` or another?

Reply:
`test_channel_model.py` seems the best place for this.